lib/shelley/src/Cardano/Wallet/Shelley/Launch.hs

```diff
@@ -965,7 +1217,16 @@ submitTx tr name signedTx = do
     void $ cliRetry tr ("Submitting transaction for " ++ name)
         [ "shelley", "transaction", "submit"
         , "--tx-file", signedTx
-        , "--mainnet"
+        , "--mainnet", "--cardano-mode"
+        ]
+
+-- | Submit a transaction through a running node.
+submitTxNoRetry :: Tracer IO ClusterLog -> FilePath -> IO ()
+submitTxNoRetry tr signedTx = do
```

Comment:
What's the use for this function? Why would we want to drop the retry behavior?

Reply:
🤔 Yeah, you're right.